connector/src/main/java/org/geysermc/connector/network/translators/inventory/updater/ChestInventoryUpdater.java

```diff
@@ -50,7 +47,9 @@ public void updateInventory(InventoryTranslator translator, GeyserSession sessio
             if (i < translator.size) {
                 bedrockItems[i] = GeyserEdition.TRANSLATORS.getItemTranslator().translateToBedrock(session, inventory.getItem(i));
             } else {
-                bedrockItems[i] = UNUSUABLE_SPACE_BLOCK;
+                bedrockItems[i] = GeyserEdition.INVENTORY_UTILS.createUnusableSpaceBlock(
+                        "This slot does not exist in the inventory\non Java Edition, as there is less\nrows than possible in Bedrock"
+                );
```

Comment:
This should stay how it was so we are not creating a new object every time

Reply:
Ditto for this.